content/kubermatic/master/architecture/concept/kkp-concepts/kkp-security/pod-security-policy/_index.en.md

```diff
@@ -11,6 +11,12 @@ This is also true for existing clusters. Without an authorizing policy, the cont
 
 PSP objects are cluster-level objects. They define a set of conditions that a pod must pass to be accepted by the PSP admission controller. The most common way to apply this is using RBAC. For a pod to use a specific Pod Security Policy, the pod should run using a Service Account or a User that has `use` permission to that particular Pod Security policy.
 
+## Note on PodSecurityPolicy deprecation
+
+Please note that PodSecurityPolicy [Pod Security Policy deprecation](https://kubernetes.io/docs/concepts/security/pod-security-policy/) was deprecated in Kubernetes v1.21, and removed from Kubernetes in v1.25.
+
+We suggest to move already to [Open Policy Agent (OPA)](https://docs.kubermatic.com/kubermatic/master/tutorials-howtos/opa-integration/opa-integration-via-ui/) as a replacement.
```

Comment:
The link should be a relative reference so it links to the version of this document for the respective KKP version.

A minor grammar nitpick, I think the "already" in this sentence can be removed, but that is probably just personal taste.

Reply:
👍